src/modules/cxxpiper.cpp

```diff
@@ -0,0 +1,1223 @@
+
+#include <sys/time.h>
+#include <array>
+#include <chrono>
+#include <condition_variable>
+#include <fstream>
+#include <functional>
+#include <limits>
+#include <filesystem>
+#include <sstream>
+#include <mutex>
+#include <stdexcept>
+#include <vector>
+
+#include <espeak-ng/speak_lib.h>
+#include <onnxruntime_cxx_api.h>
+#include "json.hpp"
+#include "piper.hpp"
+#include "utf8.h"
+#include "wavfile.hpp"
+#include "../rubberband/RubberBandStretcher.h"
```

Comment:
A `-I/where/rubberband/is` should be added to `Makefile.am` instead. It's available as system-provided packages in Debian for instance.